template_content/playground/state/state_demo.py

```diff
@@ -0,0 +1,54 @@
+from algokit_utils import LogicError
+import application
```

Comment:
I think we should standardize what to call files that have a smart contract in them. 
Should we say "application.py" for all smart contract files? If so, what should we do if there are multiple smart contracts for a project? Or should we name the file to match the smart contract name?

Reply:
I was just thinking about this,  I'm not sure what the right answer is but it seems reasonable to call the file that holds the `app = Application...` to be `application.py`